pkgdb2client/cli.py

```diff
@@ -781,11 +778,8 @@ def main():
         LOG.setLevel(logging.INFO)
 
     if arg.test:
-        global fasclient, pkgdbclient
+        global pkgdbclient
         print "Testing environment"
-        fasclient = AccountSystem(
```

Comment:
Same here.  It would be nice to be able to pass URLs and ``insecure={True,False}`` to be able to use this in staging or a local box (or some place else).